deps.edn

```diff
@@ -21,4 +21,12 @@
                  :git/tag "v0.5.1"
                  :git/sha "dfb30dd"}}
    :main-opts ["-m" "cognitect.test-runner"]
-   :exec-fn cognitect.test-runner.api/test}}}
+   :exec-fn cognitect.test-runner.api/test}
+
+  :dev
+  {:extra-paths ["test"]
+   :extra-deps {io.github.cognitect-labs/test-runner
+                {:git/url "https://github.com/cognitect-labs/test-runner"
+                 :git/tag "v0.5.1"
+                 :git/sha "dfb30dd"}}}
```

Comment:
I'll delete this if we don't want the extra alias. I added it to have a way to get an editor-connected REPL with test dependencies.